### PR TITLE
how to best revert documentation

### PR DESCRIPTION
**Pull request by cameroncaci**
Given recent reverts to main and main locks, this outlines the best way to reduce conflicts with reverting from main when no feature flags are able to be utilized.

### CODE REVIEW

**Review by deandreJones — APPROVED** [review]
I can't spell.. but looks good to me

File: docs/tools/github/Reverting-in-main-with-an-integration-branch.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## How does this work
+If you were to revert from main first and not integration under the assumption that since it's in integration it's fine, it doesn't need to be reverted there, then the moment another developer merges their main branch into their integration branch things will begin to fall apart. At worst it will pull in your revert and break everything in integration, or at best cause annoying merge conflicts not just for one developer, but any and all developers actively merging main to integration before as resolution has been pushed.
```
Comment (traskowskycaci) [inline docs/tools/github/Reverting-in-main-with-an-integration-branch.md:15]: One small typo - "`before as resolution`"

Comment (cameroncaci) [reply docs/tools/github/Reverting-in-main-with-an-integration-branch.md]: Resolved in [1e7f7db](https://github.com/transcom/mymove-docs/pull/550/commits/1e7f7db4216a0ee811843bfa949e9a1e59e054d7)

**Review by traskowskycaci — APPROVED** [review]
Looks good! Thanks for this writeup

**Review by danieljordan-caci — APPROVED** [review]
Ooooooo niiiiice. Love this. Good work!

**Pull request merged**